### ISSUE TITLE
Nuke 8.4.1 Carthage installation problem

### ISSUE THREAD

**Issue by b-onc** [kean/Nuke#360]
In our example project (https://github.com/GetStream/stream-chat-swift) we use Nuke 8.4.1 via Carthage. Installation works for some people and fails for some others, we could not pinpoint the exact issue so we created a simple project with just Nuke dependency and tried again.

Cartfile:
```
github "kean/Nuke" ~> 8.4
```
Cartfile.resolved:
```
github "kean/Nuke" "8.4.1"
```
Terminal:
```
➜  CarthageDemo git:(master) ✗ rm -rf ~/Library/Caches/org.carthage.CarthageKit
➜  CarthageDemo git:(master) ✗ rm -rf Carthage
➜  CarthageDemo git:(master) ✗ carthage update --platform iOS
*** Cloning Nuke
*** Checking out Nuke at "8.4.1"
*** xcodebuild output can be found in /var/folders/ld/9gp2554s1nn7f25tlpb0rfyh0000gn/T/carthage-xcodebuild.FVlpra.log
*** Skipped downloading Nuke.framework binary due to the error:
	"API rate limit exceeded for 77.173.173.26. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)"
A shell task (/usr/bin/xcrun xcodebuild -project /Users/bahadir/Downloads/CarthageDemo/Carthage/Checkouts/Nuke/Demo/NukeDemo.xcodeproj CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES -list) failed with exit code 74:
xcodebuild: error: Could not resolve package dependencies:
  Authentication failed because no credentials were provided.
```

(Please disregard rate limit skip, it's the same without rate limit)

Without any changes, we just changed our Cartfile to
```
github "kean/Nuke" == 8.3.1
```
and Carthage succeded.

Any idea what might be causing this? Especially strange since it works for some people without issues.

**Comment by kean** [comment]
I'm not entirely sure what's going on there. I tested 8.4.1 prior to releasing it, and I tested it now just in case. It works fine. 

I would suggest to make sure you are using the supported Xcode version (and also the correct version of Command Line Tools). I also use the latest Carthage (0.34).

**Comment by b-onc** [comment]
We're using latest version of Carthage, with latest Xcode (11.4_11E146) and latest CL tools.
When it works (for some of us), we don't see the Demo project building in logs.
Do you have any pointers, stuff we should check?

**Comment by kean** [comment]
Are you explicitly building the demo project? I did make some change to the demo in Nuke 8.4.0, I removed Carthage and updated it to use Swift Package Manager. These changes should not have affected users of the framework in any way, as demo is in its own folder and doesn't have any shared schemes. Unless you are explicitly trying to build it, you should be fine.

**Comment by b-onc** [comment]
No, we weren't building the demo project, we just ran 
`carthage update --platform iOS --new-resolver`

Interestingly, it seems like a problem with our github auth, in which --verbose option did not help us. Re-adding github account and removing gitconfig file seems to have resolved this issue but we can't pinpoint where exactly.

What's bizarre is in our Example project, Carthage was building Gzip before Nuke and that did not fail. That's why I opened this issue here and not in Carthage repo, suspecting issue came from Nuke.

Since we can't reproduce anymore I'm going to close this issue. Thanks for the help!
